src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecoratorTest.java

```diff
@@ -157,8 +158,8 @@ public void testQuietCommandExecution() throws Exception {
 
     @Test
     public void testCommandExecutionWithNohup() throws Exception {
-        ProcReturn r = execCommand(false, "nohup", "sh", "-c", "cd /tmp; echo pid is $$$$ > test; cat /tmp/test");
-        assertFalse("Output should not contain pid: " + r.output, PID_PATTERN.matcher(r.output).find());
```

Comment:
how can you test that it is actually executed with nohup?

Reply:
Good point. I didn't realize that the assertion was actually using the side-effect of output being redirected to `nohup.out` to check that nohup was actually called.

I will check if it can be asserted in another way.

Reply:
I could check if `sh` was called using `nohup` by actually sending a `HUP` to the shell itself: 

```java
      ProcReturn r = execCommand(false, "nohup", "sh", "-c", "kill -1 $$$$; sleep 5; cd /tmp; echo pid is $$$$ > test; cat /tmp/test");
        assertTrue("Output should contain pid: " + r.output, PID_PATTERN.matcher(r.output).find());
```

When starting with nohup, `echo pid is $$` will be executed, without nohup it won't.

The problem is that it does not work in busybox, only using "real shells" like bash or dash - seems that busybox somehow resets the SIGHUP handlers when starting a shell, which might be a problem even outside of the tests - although I didn't observe any problems yet..

Reply:
I thought about this some more, and it seems to me that the assertion provides little value.

We know that the commands are executed using `nohup`, as the whole point of JENKINS-46087 is that `nohup.out` is created because of it (which we want to prevent).

As far as I know, the output that is streamed via websocket is not used anyway, but written to `durable-*/jenkins-log.txt` in the workspace:

```
Executing command: "nohup" "sh" "-c" "echo \$\$ > '/home/jenkins/workspace/full@tmp/durable-ceddedcc/pid'; jsc=durable-a6507f883413e029daba773be43aa135; JENKINS_SERVER_COOKIE=\$jsc '/home/jenkins/workspace/full@tmp/durable-ceddedcc/script.sh' > '/home/jenkins/workspace/full@tmp/durable-ceddedcc/jenkins-log.txt' 2>&1; echo \$? > '/home/jenkins/workspace/full@tmp/durable-ceddedcc/jenkins-result.txt'" 
printf "EXITCODE %3d" $?; exit
```

This is where durabletask picks it up.

Do you need any new tests verifying the new behavior?